apps/src/lab2/views/components/editor/editorConfig.ts

```diff
@@ -14,7 +14,6 @@ import {
 } from '@codemirror/commands';
 import {
   indentOnInput,
-  foldGutter,
```

Comment:
Sad to see you go, foldGutter!
Perhaps we can submit an issue? https://github.com/codemirror/dev/issues